cve/system/ksp-cve-2021-41277-metabase-file-inclusion.yaml

```diff
@@ -0,0 +1,50 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-cve-2021-41277-metabase-file-inclusion
+  namespace: default # Change to your namespace
+spec:
+  tags: ["CVE", "Metabase", "File Inclusion", "CVE-2021-41277"]
+  message: "Auditing Metabase access"
+  selector:
+    matchLabels:
+      app: metabase  #Change to labels of frontend web application pod 
+  file:
+    severity: 5
+    matchPatterns:
+    - pattern: /**/*
```

Comment:
This will block everything, is there any constraints as well?

Reply:
This policy will allow only to application run, here the custom GeoJSON which is supported and potential local file inclusion is blocked. me and Vishnu have tested this policy.